charts/kubearchive/templates/api_server/api_server.yaml

```diff
@@ -36,6 +43,16 @@ spec:
               value: "disabled"
             - name: OTEL_EXPORTER_OTLP_ENDPOINT
               value: ""
+            - name: OTEL_GO_X_DEPRECATED_RUNTIME_METRICS
+              value: "false"
+            - name: GOMEMLIMIT
+              valueFrom:
+                resourceFieldRef:
+                  resource: limits.memory
```

Comment:
[The Go Garbage Collector Documentation](https://tip.golang.org/doc/gc-guide#Memory_limit) has this to say about setting memory limits when deploying a go application in a container

```
a good rule of thumb is to leave an additional 5-10% of headroom to account for memory sources the Go runtime is unaware of
```

based this, we might wanna set the value as 90%-95% of `limits.memory` instead

Reply:
Specifying it this way is very convenient... What about setting  `requests.memory` to 90% and use it to set the GOMEMLIMIT? That would change the QoS class of the pod to Burstable, but I think we can manage with that.

Reply:
I think that will work :+1:

Reply:
Done